src/firefly/js/visualize/ui/CatalogSelectViewPanel.jsx

```diff
@@ -0,0 +1,623 @@
+/*
+ * License information at https://github.com/Caltech-IPAC/firefly/blob/master/License.txt
+ */
+
+import React, {Component, PropTypes} from 'react';
+import FormPanel from '../../ui/FormPanel.jsx';
+import { get, merge, isEmpty} from 'lodash';
+import {updateMerge} from '../../util/WebUtil.js';
+import {ListBoxInputField} from '../../ui/ListBoxInputField.jsx';
+import {TableRequest} from '../../tables/TableRequest';
+import * as TblUtil from '../../tables/TableUtil.js';
```

Comment:
You are only imported two function you probably should reference the directly.
```
import {uniqueTblUiGid, doFetchTable} from '../../tables/TableUtil.js';
```
remember to change the code that uses it as well.